contracts/protocol/libraries/logic/SupplyLogic.sol

```diff
@@ -167,21 +166,28 @@ library SupplyLogic {
         DataTypes.UserConfigurationMap storage userConfig,
         DataTypes.ExecuteSupplyERC721Params memory params
     ) external {
+        DataTypes.ReserveData storage reserve = reservesData[params.asset];
+        DataTypes.ReserveCache memory reserveCache = reserve.cache();
```

Comment:
we acgtually dont need to perform the .cache() operation since its too expensive. we can just pass the parameters that are needed. I think we only need the configuration, ntokenAddress and reserveId?

Reply:
yeah. it's ture.